account.js

```diff
@@ -0,0 +1,104 @@
+var readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+class BankAccount {
+  constructor(name, accountNumber, phoneNumber, balance, accountStatus) {
+    this.balance = balance;
+    this.name = name;
+    this.accountNumber = accountNumber;
+    this.phoneNumber = phoneNumber;
+    this.accountStatus = accountStatus;
+  }
+
+  open() {
```

Comment:
I don't see the significance of this method. Generally, this whole question doesn't make sense to me. Why would an account open its self. I wish they told you guys to implement a `Bank` and a `BankAccount` class.
The only way this can make sense is if the account has three states i.e `CREATED`, `ACTIVE` and `CLOSED`. The open method would set the state to `ACTIVE` and you wouldn't be able to `close`, `withdraw`, `deposit` or `getBalance` from an account that is not `ACTIVE`